app/src/main/res/layout/widget_media_controls.xml

```diff
@@ -0,0 +1,135 @@
+                    android:padding="4dp"
+                    android:contentDescription="@string/widget_media_prev_track_button"
+                    android:src="@drawable/ic_prev_track"
+                    android:scaleType="fitCenter"
+                    android:background="?android:selectableItemBackground"/>
+
+                <ImageButton
+                    android:id="@+id/widgetRewindButton"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:padding="4dp"
+                    android:contentDescription="@string/widget_media_rewind_button"
+                    android:src="@drawable/ic_rewind"
+                    android:scaleType="fitCenter"
+                    android:background="?android:selectableItemBackground"/>
+
+                <ImageButton
+                    android:id="@+id/widgetPlayPauseButton"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1.5"
+                    android:padding="4dp"
+                    android:contentDescription="@string/widget_media_playpause_button"
+                    android:src="@drawable/ic_playpause"
+                    android:scaleType="fitCenter"
+                    android:background="?android:selectableItemBackground"/>
+
+                <ImageButton
+                    android:id="@+id/widgetFastForwardButton"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:padding="4dp"
+                    android:contentDescription="@string/widget_media_fastforward_button"
+                    android:src="@drawable/ic_fastforward"
+                    android:scaleType="fitCenter"
+                    android:background="?android:selectableItemBackground"/>
+
+                <ImageButton
+                    android:id="@+id/widgetNextTrackButton"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:padding="4dp"
+                    android:contentDescription="@string/widget_media_next_track_button"
+                    android:src="@drawable/ic_next_track"
+                    android:scaleType="fitCenter"
+                    android:background="?android:selectableItemBackground"/>
+            </LinearLayout>
+        </RelativeLayout>
+    </LinearLayout>
+
+    <ProgressBar
+        android:id="@+id/widgetProgressBar"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:padding="8dp"
+        android:visibility="invisible"
+        android:indeterminate="true"
+        android:indeterminateTint="@color/colorAccent"
+        />
```

Comment:
Does this ever become active?

Reply:
Uhhh...don't mind me, just leaving in code I copy-pasted from my last widget...

That's not supposed to be there, sorry. I'll remove it.

Reply:
Removed in e3a66e5